src/services/TaggedImages.js

```diff
@@ -35,15 +36,17 @@ import client, { prefixPath } from './DavClient.js'
 export default async function(id, options = {}) {
 
 	options = Object.assign({
-		method: 'REPORT',
+		headers: {
+			method: 'REPORT',
```

Comment:
`method` is in the `headers` object?

Reply:
Yes for our patched client from `@nextcloud/files` otherwise we would need to use `customRequest` and manually parse the returned XML.